ArcTouchChallenge/ArcTouchChallenge/AppDelegate.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  AppDelegate.swift
+//  ArcTouchChallenge
+//
+//  Created by Arthur Augusto Sousa Marques on 3/13/18.
+//  Copyright © 2018 Arthur Augusto. All rights reserved.
+//
+
+import UIKit
+
+let sharedAppDelegate: AppDelegate = (UIApplication.shared.delegate as! AppDelegate)
```

Comment:
I've made a general instance of the AppDelegate, but it could be a static var inside the class. Both make the life of the variable last until the program is terminated and makes it accessible in any part of the code, but I do not have a well-defined decision.